src/lib/_common.scss

```diff
@@ -155,6 +155,11 @@ $header-height: 48px;
         position: absolute;
         right: 0;
         top: 0;
+
+        :first-child:focus {
```

Comment:
Is this generating `*:first-child:focus`? If so, can we put the asterisk in here explicitly? You also should consider scoping it more tightly:

```
> *[tab-index="0"]:first-child:focus {
    box-shadow: inset 0 0 7px 0 fade-out($twos, .75);
    outline: 0;
}
```

Reply:
per our discussion offline I'm going to switch to a class to account for CSVs